torchio/transforms/augmentation/composition.py

```diff
@@ -106,3 +124,21 @@ def _normalize_probabilities(transforms_dict: dict):
             raise ValueError(message)
         for transform, probability in transforms_dict.items():
             transforms_dict[transform] = probability / probabilities.sum()
+
+
+def compose_from_history(history: List):
+    """
+    Builds a composition of transformations from a given subject history
+    :param history: subject history given as a list of tuples containing (transformation_name, transformation_parameters)
+    :return: Tuple (Compose of transforms, list of seeds to reproduce the transforms from the history)
+    """
```

Comment:
Can you switch to Google style instead of NumPy style?

https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html